src/users/login.js

```diff
@@ -57,10 +59,20 @@ export default class Login extends Component {
 
   onSubmit = (e) => {
     e.preventDefault();
-    localStorage.setItem('student_id', this.state.studentID);
+
+    const user = {
+      email: this.state.email,
+      password: this.state.password
+    }
+
+    login(user).then(res => {
+      this.props.history.push('/dnd');
+    })
+
+    /*localStorage.setItem('email', this.state.email);
     localStorage.setItem('password', this.state.password);
     localStorage.setItem('loggedIn', true);
-    window.location.href = '/dnd';
```

Comment:
need this to reroute user to main page or have error message show if cannot login. right now it doesn't show anything for me